presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveUtil.java

```diff
@@ -228,7 +228,15 @@ private HiveUtil()
                 .filter(name -> name.startsWith("serialization."))
                 .forEach(name -> jobConf.set(name, schema.getProperty(name)));
 
-        configureCompressionCodecs(jobConf);
+        // add Airlift LZO and LZOP to head of codecs list so as to not override existing entries
+        List<String> codecs = newArrayList(Splitter.on(",").trimResults().omitEmptyStrings().split(jobConf.get("io.compression.codecs", "")));
```

Comment:
Revert commit is here only to make sure the test fails before the change and passes afterwards.